C7/Game.cs

```diff
@@ -171,6 +171,15 @@ public void processEngineMessages(GameData gameData) {
 					break;
 				case MsgCityDestroyed mCD:
 					mapView.cityLayer.UpdateAfterCityDestruction(mCD.city);
+
+					// If this was the last city of the civilization, display a popup
+					// noting that the civ is gone and destroy any remaining units.
+					if (mCD.city.owner.RemainingCities() == 0) {
```

Comment:
If I recall correctly, in Civ 3 a civilization will stay alive as long as it has at least one settler, even if all its cities are destroyed.

Reply:
Yes ultimately the game conditions decide this, i.e. 1 city elimination, regicide, etc. For now this may be a good starting place until we implement game rules.

Reply:
Agreed. I've added a TODO to note this discrepancy, but I think this is still worth doing.

Adding the settler check means we also need to run this type of check every time a unit is destroyed, in case it was the last settler, and this at least gives us a starting point.

Reply:
Agreed, this is good enough for a first pass but can be extracted to an event later with all the proper checks